libgnucash/backend/xml/gnc-xml-backend.hpp

```diff
@@ -60,7 +60,7 @@ class GncXmlBackend : public QofBackend
     std::string m_dirname;
     std::string m_lockfile;
     std::string m_linkfile;
-    int m_lockfd;
+    int m_lockfd = 0;
```

Comment:
No necessary, the compiler does that, see[item 3 under Explanation here](https://en.cppreference.com/w/cpp/language/default_initialization).

Reply:
so the answer is https://en.cppreference.com/w/cpp/language/zero_initialization